src/html.rs

```diff
@@ -38,9 +38,10 @@ pub fn rewrite_html_base_url(
     cdn_url: Option<&str>,
 ) -> Result<Vec<u8>> {
     let rewrite_url_in_attr = |el: &mut Element, attr_name: &str| {
-        if let Some(attr) = el.get_attribute(attr_name) {
+        if let Some(mut attr) = el.get_attribute(attr_name) {
             let mut base_url = "";
             if attr.starts_with("/static") {
+                attr = attr.replacen("/static", "", 1);
```

Comment:
We can use `strip_prefix()`:

```rust
if let Some(attr) = attr.strip_prefix("/static") {

}
```

Reply:
please https://github.com/zineland/zine/pull/140/commits/3672c321ba8576c47dbf488bb2abcf5e60a15c3b